common-libs/internal-api-clients/src/spec/_keyStorage.ts

```diff
@@ -265,10 +188,14 @@ export default {
 	},
 	"info": {
 		"title": "affinity-wallet-backend",
-		"version": "1.0.58",
+		"version": "1.4.0",
```

Comment:
Pls set latest version of [spec](https://gitlab.com/affinidi/foundational/affinity-wallet-backend/-/blob/main/swagger.json#L190) for affinity-wallet-backend

Reply:
ok